hack/jenkins/e2e-runner.sh

```diff
@@ -192,6 +192,7 @@ function e2e_test() {
         echo "Publish version to ci/latest-green.txt: $(cat version)"
         gsutil cp ./version gs://kubernetes-release/ci/latest-green.txt
     fi
+    return ${exitcode}
```

Comment:
I guess that was already triggering a complete test failure? Either that or it wasn't causing a failure at all?